local/Dockerfile.ubi8

```diff
@@ -0,0 +1,87 @@
+#
```

Comment:
from what i can tell there are only three differences between this dockerfile and the dockerfile for cp-kafka. 

1. the base image
2. you install rest-proxy
3. you expose an additional port

Can you find a way to re-use the cp-kafka dockerfile and just add the two things you need and base it on a different image instead of copying the entire docker file?